resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/DriverServiceFeatureStep.scala

```diff
@@ -55,6 +56,9 @@ private[spark] class DriverServiceFeatureStep(
   private val driverBlockManagerPort = kubernetesConf.sparkConf.getInt(
     config.DRIVER_BLOCK_MANAGER_PORT.key, DEFAULT_BLOCKMANAGER_PORT)
 
+  val driverUIPort = SparkUI.getUIPort(kubernetesConf
+    .sparkConf)
```

Comment:
fits in previous line. but really new code should use `sparkConf.get(UI_PORT)`.

Reply:
Made these changes and pushed.